doc/projects_list.rst

```diff
@@ -0,0 +1,26 @@
+=======================
+Who uses Sphinx-Gallery
+=======================
+
+Here is a list of projects using `sphinx-gallery`.
+
+* `Sphinx-Gallery <https://sphinx-gallery.github.io/en/latest/auto_examples/index.html>`_
+* `Scikit-learn <http://scikit-learn.org/dev/auto_examples/index.html>`_
+* `Nilearn <https://nilearn.github.io/auto_examples/index.html>`_
+* `MNE-python <https://www.martinos.org/mne/stable/auto_examples/index.html>`_
+* `PyStruct <https://pystruct.github.io/auto_examples/index.html>`_
+* `GIMLi <http://www.pygimli.org/_examples_auto/index.html>`_
+* `Nestle <https://kbarbary.github.io/nestle/examples/index.html>`_
+* `pyRiemann <https://pythonhosted.org/pyriemann/auto_examples/index.html>`_
```

Comment:
this one didn't work for me :-/

Reply:
It's https://pyriemann.readthedocs.io/en/latest/index.html but the package does not seem to be updated any more, maybe we should just remove it.